src/Server/LspRequestRouter.cs

```diff
@@ -121,10 +121,12 @@ private ILspHandlerDescriptor FindDescriptor(string method, JToken @params)
                     object @params;
                     try
                     {
-                        @params = request.Params.ToObject(descriptor.Params, _serializer.JsonSerializer);
+                        @params = request.Params?.ToObject(descriptor.Params, _serializer.JsonSerializer);
```

Comment:
BTW, am wondering if perhaps `descriptor.Params` should be `descriptor.ParamsType`?